.github/workflows/build.yml

```diff
@@ -30,30 +30,30 @@ jobs:
           flutter-version: ${{ env.FLUTTER_VERSION }}
 
       # API
-      - name: "api: Install dependencies"
+      - name: 'api: Install dependencies'
         working-directory: ./api
         run: dart pub get
-      - name: "api: Run code generator(s)"
+      - name: 'api: Run code generator(s)'
         working-directory: ./api
         run: dart pub run build_runner build
 
-      - name: "api: Remove obsolete files"
+      - name: 'api: Remove obsolete files'
         working-directory: ./api
         run: rm -rf ./.dart_tool/pub/bin/build_runner
 
       # App
-      - name: "app: Install dependencies"
-        working-directory: ./app
-        run: flutter pub get
-      - name: "app: Run code generator(s)"
-        working-directory: ./app
-        run: |
-          flutter pub run intl_utils:generate
-          flutter pub run build_runner build
+      # - name: "app: Install dependencies"
+      #   working-directory: ./app
+      #   run: flutter pub get
+      # - name: "app: Run code generator(s)"
+      #   working-directory: ./app
+      #   run: |
+      #     flutter pub run intl_utils:generate
+      #     flutter pub run build_runner build
 
-      - name: "app: Remove obsolete files"
-        working-directory: ./app
-        run: rm -rf ./.dart_tool/pub/bin/build_runner
+      # - name: "app: Remove obsolete files"
+      #   working-directory: ./app
+      #   run: rm -rf ./.dart_tool/pub/bin/build_runner
```

Comment:
Why is this commented out?

Reply:
The app itself is not yet migrated to null-safety but the whole build-script only specifies a single Flutter version. And if we focus on the API wrappers for now and then migrate the app, we don't really need it to be checked at the moment.